### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by swatisehgal**
- Capture steps to configure and run topology-aware scheduler plugin

Signed-off-by: Swati Sehgal <swsehgal@redhat.com>

### CODE REVIEW

**Comment by swatisehgal** [comment]
This might need updates when we are able to get the latest version of the out of tree scheduler working. Currently the scheduler plugin is being debugged.

File: pkg/noderesourcetopology/README.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+### Config
+ filter must be enabled for topology-aware scheduling.
+```
+apiVersion: kubescheduler.config.k8s.io/v1beta1
+kind: KubeSchedulerConfiguration
+leaderElection:
+  leaderElect: false
+clientConnection:
+  # kubeconfig: "REPLACE_ME_WITH_KUBE_CONFIG_PATH"
```
Comment (AlexeyPerevalov) [inline pkg/noderesourcetopology/README.md]: I think we have to leave only one kubeconfig. I think it's better to keep
"/etc/kubernetes/scheduler.conf". I see it derives from examples.

**Review by AlexeyPerevalov — CHANGES_REQUESTED** [review]
Other than kubeconfig looks good for me)

File: pkg/noderesourcetopology/README.md (hunk 1 above)
Comment (swatisehgal) [reply pkg/noderesourcetopology/README.md]: Sure, will update

Comment (swatisehgal) [reply pkg/noderesourcetopology/README.md]: Done!

**Review by AlexeyPerevalov — APPROVED** [review]

**Pull request merged**